src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -1299,6 +1314,8 @@ contract StrategyManagerUnitTests is Test {
     }
 
     function testSlashSharesNotBeaconChainETHFuzzed(uint64 withdrawalAmount) external {
+        // cannot cause share value to increase too drastically
+        cheats.assume(withdrawalAmount <= 1e9 || withdrawalAmount == 1e18);
```

Comment:
this is some weird filtering -- does the test just make really unusual assumptions about the starting state of the storage?

Reply:
I still don't know what's happening here

Reply:
thanks